src/systems/frequency_system.C

```diff
@@ -208,20 +208,20 @@ void FrequencySystem::set_frequencies_by_steps (const Real base_freq,
   _finished_set_frequencies = true;
 
   // set the current frequency
-  this->set_current_frequency(0);
+  this->set_current_frequency(0.);
 }
 
 
 
-void FrequencySystem::set_frequencies_by_range (const Real min_freq,
-                                                const Real max_freq,
+void FrequencySystem::set_frequencies_by_range (const Number min_freq,
+                                                const Number max_freq,
                                                 const unsigned int n_freq,
                                                 const bool allocate_solution_duplicates)
 {
   this->_keep_solution_duplicates = allocate_solution_duplicates;
 
-  // sanity checks
-  libmesh_assert_greater (max_freq, min_freq);
+  // sanity checks. Only look at the real part.
+  libmesh_assert_greater (std::real(max_freq), std::real(min_freq));
```

Comment:
Should this be libmesh_assert_greater_equal now?  Could users want to step through different damping levels at the same frequency?